src/test-exercice/calculator.js

```diff
@@ -0,0 +1,18 @@
+const Calculator = {
+  addition(num1, num2) {
+    return num1 + num2;
+  },
+
+  subtraction(num1, num2) {
+    return num1 - num2;
+  },
+
+  division(num1, num2) {
+    return num1 / num2;
+  },
+
+  multiplication(num1, num2) {
+    return num1 * num2;
+  },
+};
+module.exports = Calculator;
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you remove the `test-exercise` directory with its content as it has files non-related to the `Todo` project